EIPS/eip-3450.md

```diff
@@ -0,0 +1,151 @@
+
+1. Convert each share mnemonic to a value-vector, where each value is the index of the corresponding word in the word list
+2. Define values:
+   - Let _y<sub>k</sub>_ be the value-vector of share _k_
+   - Let _x<sub>k</sub>_ be the id of share _k_
+3. Calculate _E_ = Interpolate(0, [(_x<sub>1</sub>_, _y<sub>1</sub>_),...,(_x<sub>K</sub>_, _y<sub>K</sub>_)])
+   - See [Polynomial Interpolation](#polynomial-interpolation) above
+4. Convert _E_ to a mnemonic using each value to lookup a word by index in the word list
+
+## Test Cases
+
+[Test vectors](https://github.com/danielstreit/shamir-bip-39/blob/272333114ce03dfb5f4ed94f7ddb22e7cc0c93c4/test/vectors.ts)
+
+All implementations must be able to:
+
+- Split and recover each `mnemonic` with the given `numShares` and `threshold`.
+- Recover the `mnemonic` from the given `knownShares`.
+
+## Reference Implementation
+
+https://github.com/danielstreit/shamir-bip39
+
+## Security Considerations
+
+The shares produced by the specification include an id in addition to the BIP-39 mnemonic. This raises two security concerns:
+
+Users **must** keep this id in order to recover the original mnemonic. If the id is lost, or separated from the share mnemonic, it may not be
+possible to recover the original. (Brute force recovery may or may not be possible depending on how much is known about the number of shares and threshold)
+
+The additional data may hint to an attacker of the existence of other keys and the scheme under which they are stored. Ideally,
+if we could only store the BIP-39 mnemonic, this would be indistinguishable from any other BIP-39 mnemonic and an attacker may not
+look for additional shares. If an attacker recognized that the BIP-39 mnemonic came with an id, it might hint to them that there
+are additional shares.
+
+A potential alternative specification might include a "digest" in the polynomial that could be used to validate the recovered secret. See, for example,
+[SLIP-0039](https://github.com/satoshilabs/slips/blob/master/slip-0039.md#shamirs-secret-sharing).
+If we combined this validation with a limited share count so that we could brute force the ids by attempting every permutation of
+shares and ids until we recovered a valid mnemonic, we could avoid storing the id.
+
+We decided against this approach and in favor of storing the id because, while the id may leak some information, recovery validation also leaks information.
+Without validation, an attacker may never know when they have sufficient shares for recovery. With validation, the status of recovery is known.
+
+To mitigate leaking information, users can consider using an alias for ids or obscuring them in some manner.
+
+## References and Credits
+
+This specification leans heavily on prior work, including:
+
+- The [SLIP-0039 Specification](https://github.com/satoshilabs/slips/blob/master/slip-0039.md)
+- Grempe's [secrets.js](https://github.com/grempe/secrets.js) implementation of Shamir's Secret Sharing Scheme
+
```

Comment:
```suggestion
```
This is not a valid EIP section.  Also, EIPs are CC0 which does not require attribution.  And of course, no external links.  😉

Reply:
Removed.